### PR TITLE
Fixes delaying round start not showing on the timer.

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request

As title

# Explain why it's good for the game

It's not always clear if it's been delayed or somehow broken anymore.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
fix: Delaying round start now shows on the timer again.
/:cl:

### CODE REVIEW

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -1189,7 +1189,7 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	if(!SSticker.HasRoundStarted())
 		var/time_remaining = SSticker.GetTimeLeft()
 		if(time_remaining > 0)
-			. += "Time To Start: [round(time_remaining)]s"
+			. += "Time To Start: [round(time_remaining)]s [SSticker.delay_start ? "(DELAYED)" : ""]"
```
Comment (Drulikar) [inline code/modules/mob/dead/observer/observer.dm:1192]:
```suggestion
			. += "Time To Start: [round(time_remaining)]s[SSticker.delay_start ? " (DELAYED)" : ""]"
```

**Review by Drulikar — CHANGES_REQUESTED** [review]
You likely want to also change `/mob/new_player/get_status_tab_items()` too then because at the moment, this only affects admins who observe before game start.

**Comment by realforest2001** [comment]
No idea how I missed new_player

File: code/modules/mob/new_player/new_player.dm [hunk 2]
```diff
@@ -451,7 +451,7 @@
 
 	var/time_remaining = SSticker.GetTimeLeft()
 	if(time_remaining > 0)
-		. += "Time To Start: [round(time_remaining)]s"
+		. += "Time To Start: [round(time_remaining)]s"[SSticker.delay_start ? " (DELAYED)" : ""]"
```
Comment (Drulikar) [inline code/modules/mob/new_player/new_player.dm:454]:
```suggestion
		. += "Time To Start: [round(time_remaining)]s[SSticker.delay_start ? " (DELAYED)" : ""]"
```

**Review by Drulikar — APPROVED** [review]

**Pull request merged**